common/src/main/java/com/github/khanshoaib3/minecraft_access/features/MouseKeySimulation.java

```diff
@@ -23,23 +20,14 @@
  * 4) mouse wheel scroll up<br>
  * 5) mouse wheel scroll down
  */
-@Slf4j
 public class MouseKeySimulation {
-    private static final MouseKeySimulation instance;
-
-    private boolean enabled;
+    private static boolean ENABLED;
```

Comment:
This is not a constant.
```suggestion
    private static boolean enabled;
```